ui/components/projects/pitch-deck/pitch-deck.css

```
@@ -0,0 +1,713 @@
+.pitch-deck {
+  --pitch-ink: #f7f5ef;
+  --pitch-muted: #b8b4aa;
+  --pitch-panel: #11110f;
+  --pitch-line: rgba(247, 245, 239, 0.15);
+  --pitch-accent: #f59e0b;
+  overflow: hidden;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)